libraries/migrations/0023_library_slug_unique_case_insensitive.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.15 on 2024-10-16 17:52
+
+from django.db import migrations, models
+import django.db.models.functions.text
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("libraries", "0022_delete_commitdata"),
```

Comment:
the heck is "0022_delete_commitdata" ??

Reply:
The last migration before this, created in 48c09d3d5ed6ebb